joschahenningsen/TUM-Live
# Add Poll History

### Motivation and Context
Poll Histories are awesome and useful.

### Description
- Add REST endpoint
- Add `polls` view in chat
- Use bookmarks styling for chat and polls
- Remove redundant poll html code in `chat.gohtml`

### Steps for Testing
Prerequisites:
- 1 Admin
- n Accounts
- 1 Livestream

1. Log in
2. Navigate to a Livestream
3. Create poll
4. Check if indicator works
5. Answer poll
6. Close poll
7. Results should be in the poll history
8. 🕺

### Preview 🐬


https://user-images.githubusercontent.com/29633518/217258829-d047b5d0-af7e-4bd3-87d7-aad3524d2620.mov
